Source/HelixToolkit.Wpf.SharpDX/Controls/Viewport3DX.Properties.cs

```diff
@@ -268,10 +268,10 @@ public partial class Viewport3DX
             "RenderHost", typeof(DPFCanvas), typeof(Viewport3DX), new FrameworkPropertyMetadata(null));
 
         /// <summary>
-        /// The is deferred shading enabled propery
+        /// The Render Technique property
         /// </summary>
         public static readonly DependencyProperty RenderTechniqueProperty = DependencyProperty.Register(
-            "RenderTechnique", typeof(RenderTechnique), typeof(Viewport3DX), new PropertyMetadata(Techniques.RenderPhong, (s, e) => ((Viewport3DX)s).RenderTechniquePropertyChanged()));
+            "RenderTechnique", typeof(RenderTechnique), typeof(Viewport3DX), new PropertyMetadata(null, (s, e) => ((Viewport3DX)s).RenderTechniquePropertyChanged()));

```

Comment:
Because `Techniques` is no longer available as a static class, we can't set the default here. But, the property is set during construction of the view, by binding the `SelectedItem` to the `RenderTechnique` property on the view model. As this property is set when constructing the view model, the combo boxes bound to this selected value "just work".